tools/accuracy_checker/accuracy_checker/evaluators/custom_evaluators/whisper_evaluator.py

```diff
@@ -0,0 +1,176 @@
+"""
+Copyright (c) 2024 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import os
+import re
+
+import openvino_genai as ov_genai
+from transformers import AutoModelForSpeechSeq2Seq, AutoProcessor
+from transformers.pipelines.automatic_speech_recognition import \
+    AutomaticSpeechRecognitionPipeline
```

Comment:
please make these packages optionl like inflect bellow

Reply:
Agree. I had them all in try except in initial version but then I thought that packages are so common that was no sense to import, but indeed there are checks that fails to import them though.